src/main/java/mate/academy/dto/CreateBookRequestDto.java

```diff
@@ -0,0 +1,16 @@
+package mate.academy.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class CreateBookRequestDto {
+
```

Comment:
it's decided by corporative standarts either to put blank lines here and at the end of the class or not, but it definetely should be in the same style through all the project - so choose one style and apply it